### PR TITLE
Add autocompletion to headline features

### PR DESCRIPTION
**Pull request by ofedoren**
@upadhyeammit, I'm not good at writing stuff, so please review :)

### CODE REVIEW

**Comment by upadhyeammit** [comment]
@ofedoren looks good to me, maybe @melcorr can have some suggestions?

File: _includes/manuals/3.3/1.2_release_notes.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### DSL Autocompletion in templates
+
+The template editor now has embedded DSL documentation
+(https://yourforeman/templates_doc) as autocompletion feature similarly to
```
Comment (melcorr) [inline _includes/manuals/3.3/1.2_release_notes.md]:
```suggestion
(https://yourforeman/templates_doc) as an auto-completion feature that is similar to
```

File: _includes/manuals/3.3/1.2_release_notes.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ This section will be updated prior to the next release.
...
+modern text editors or IDEs. It allows users to list all available macros whilst
+writing a new or changing an existing template. An item in the list shows the
+signature of the macro, its description and examples how to use it. Each item
```
Comment (melcorr) [inline _includes/manuals/3.3/1.2_release_notes.md]:
```suggestion
signature of the macro, its description and examples of how to use it. Each item
```

**Review by melcorr — COMMENTED** [review]
tiny comments but otherwise clear !

**Comment by ofedoren** [comment]
Thanks, @melcorr and @upadhyeammit, updated.

**Review by melcorr — APPROVED** [review]
lgtm

**Comment by upadhyeammit** [comment]
Thanks @melcorr for review and @ofedoren for note! Merging this now

**Pull request merged**